package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "instructure-ui-prototypes",
+  "description": "UI Prototypes for Instructure Product Design",
+  "private": true,
+  "author": "Instructure, Inc. Engineering and Product Design",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/instructure/instructure-ui-prototypes.git"
+  },
+  "homepage": "https://github.com/instructure/instructure-ui-prototypes#readme",
+  "bugs": {
+    "url": "https://github.com/instructure/instructure-ui-prototypes/issues"
+  },
+  "workspaces": {
+    "packages": [
+      "shared/*"
+    ]
+  },
+  "scripts": {
+    "open:sandbox": "ui-scripts open-sandbox --repo instructure-ui-prototypes --username instructure"
```

Comment:
I thought we had set up that script so that it would read the repo and username from the package.json file so that they wouldn't have to be specified here.